### PR TITLE
Assign Emission Admin - Ethereum, Arbitrum and Optimism

### PR DESCRIPTION
**Pull request by defijesus**

### CODE REVIEW

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AaveV3Optimism_AssignEmissionAdminEthereumArbitirumAndOptimism_20240229.sol [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Optimism, AaveV3OptimismAssets} from 'aave-address-book/AaveV3Optimism.sol';
+import {IEmissionManager} from '@aave/periphery-v3/contracts/rewards/interfaces/IEmissionManager.sol';
+
+/**
+ * @title Assign Emission Admin - Ethereum, Arbitirum and Optimism
+ * @author karpatkey-TokenLogic & ACI
+ * - Snapshot: TODO
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AaveV3Optimism_AssignEmissionAdminEthereumArbitirumAndOptimism_20240229.sol:11]: fill todo

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Assign Emission Admin - Ethereum, Arbitirum and Optimism"
+author: "karpatkey-TokenLogic & ACI"
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:3]: you must add a discussion link as i think uis can't handle it missing. snapshot you can leave empty.

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+## Simple Summary
+
+This enables various teams to distribute rewards across Aave v3 Ethereuim.
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:8]: > Ethereuim

😅

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+## Motivation
+
+This AIP integrates four separate ARFC that each propose granting a team the privileges to distribute there choose asset across an Aave v3 deployment. The below summaries each iniative:
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:12]:
```suggestion
This AIP integrates four separate ARFC that each propose granting a team the privileges to distribute there chosen asset across an Aave v3 deployment. The below summaries each initiative:
```

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+The `Emission_Admin` can distribute the specified token anywhere across the Aave v3 deployment on the respective Liquidity Pool.
+
+## Implementation
+
+Set `Emission_Admin` permission for ETHx, SD, SWISE, osETH, OP and ARB to the following address:
+
+**Aave v3 Ethereum**
+
+Address Reward (SD): `0x30D20208d987713f46DFD34EF128Bb16C404D10f`
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:37]: as always, better do explorer links.

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+Address Reward (ARB): `0x912CE59144191C1204E64559FE8253a0e49E6548`
+EMISSION_ADMIN: `0xE79C65a313a1f4Ca5D1d15414E0c515056dA90b4`
+
+**Aave v3 Optimism**
+
+Address Reward (OP): 0x4200000000000000000000000000000000000042
+EMISSION_ADMIN: 0x3479CEb4b1fcaDC586d4c5F1c16b4d8c0D70Bc71
+
+The AIP call setEmissionAdmin() method in the emission_manager contract.
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:55]:
```suggestion
The AIP calls setEmissionAdmin() method in the EMISSION_MANAGER contract.
```

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+- OP reward via an Aave Community SAFE
+- ARB rewards via Gauntlet
+
+The two LST providers are intending on using rewards to bootstrap the growth and adoption of there resepective LST.
+
+The OP and ARB incentives are being distributed with the intent of migrating USDC.e to USDC on each respective network. These ARB and OP rewards were originally provide by the respective foundation and are not those belonging to the Aave DAO.
+
+## Specification
+
+The `EmissionManager`, a governance controlled function, assigns the `Emission_Admin` role for a specified token on the respective network.
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:25]: weird mix of writing: Now you have `EmissionManager`, `Emission_Admin` and later `EMISSION_MANAGER` ... better to just stay consistent (I'd suggest with the last one)

File: src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md [hunk 8]
```diff
@@ -22,37 +23,37 @@ The OP and ARB incentives are being distributed with the intent of migrating USD
 
 ## Specification
 
-The `EmissionManager`, a governance controlled function, assigns the `Emission_Admin` role for a specified token on the respective network.
+The `setEmissionAdmin`, a governance controlled function, assigns the `EMISSION_MANAGER` role for a specified token on the respective network.
```
Comment (sakulstra) [inline src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md:26]: i commented that you changed the casing in every sentence, now you just replaces everything with `EMISSION_MANAGER`which is wrong. E.g. here should be `EMISSION_ADMIN` like it was before.

**Review by sakulstra — APPROVED** [review]

**Review by rustboyar — APPROVED** [review]

**Pull request merged**